config/jobs/cert-manager/website/cert-manager-website-periodics.yaml

```diff
@@ -0,0 +1,34 @@
+periodics:
+
+- name: cert-manager-website-update-index
+  cluster: trusted
+  interval: 12h
+  cluster: gke
+  agent: kubernetes
+  decorate: true
+  extra_refs:
+  - org: cert-manager
+    repo: website
+    base_ref: master
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+    - image: "eu.gcr.io/jetstack-build-infra-images/golang-nodejs:20191129-c49853e-1.13.4"
+      args:
+      - bash
+      - scripts/index
+      env:
+      - name: ALGOLIA_WRITE_KEY
+        valueFrom:
+          secretKeyRef:
+            name: cert-mamager-website-algolia-key
+            key: ALGOLIA_WRITE_KEY
+      resources:
+        requests:
+          cpu: 2
```

Comment:
The VMs in this build cluster only have 2 CPUs, so probably best to set this to `cpu: 1` instead 😄 - I think you should also be able to reduce the memory request a bit? Perhaps `1Gi` or `2Gi`?

Reply:
Done, would stay at 2 since we run node.js